src/pages/IPFSPin.vue

```diff
@@ -350,6 +371,70 @@ export default {
           path: this.cid_name
         })
       }
+    },
+    async putContent (message, content, inline_requested, api_server) {
+      let inline = inline_requested
+      if (inline) {
+        let serialized = JSON.stringify(content)
+        if (serialized.length > 150000) {
+          inline = false
+        } else {
+          message.item_type = 'inline'
+          message.item_content = serialized
+          // eslint-disable-next-line no-use-before-define
```

Comment:
Why put an eslint ignore, if you changed the rule anyway?

I would either:
- Use a capitalized constructor import
- OR disable the line
- OR (worst solution) change the eslint rule

But not the 3 solutions altogether.